std/range/package.d

```diff
@@ -6590,11 +6595,53 @@ if (isForwardRange!RangeOfRanges &&
     }
 
     auto opSlice() { return this; }
+    
+    static if (hasSlicing!(ElementType!RangeOfRanges))
```

Comment:
RFC: Slicing is suddenly O(N) for a T[M][N]. Is this acceptable? Also, now that I look at the bugs again, it's not really part of the bug report. Should I remove it, file an enhancement request and make a separate PR?